src/server/server.ts

```diff
@@ -216,6 +227,33 @@ export default class Server implements Hub.RouteBuilder {
     }
   }
 
+  private async asyncProcess(req: express.Request, res: express.Response) {
+      return new Promise<void>((resolve, reject) => {
+          const child = spawn.fork(`./src/actions/actions_process.ts`)
+          child.on("message", (actionResponse) => {
+              // Some versions of Looker do not look at the "success" value in the response
+              // if the action returns a 200 status code, even though the Action API specs otherwise.
+              // So we force a non-200 status code as a workaround.
+              winston.info("Success? " + actionResponse.success + " || and what: " + JSON.stringify(actionResponse))
+              if (!actionResponse.success) {
+                  res.status(400)
+              }
+              res.json(actionResponse)
+
+              child.kill()
```

Comment:
You forgot to kill child in error state too.  Is there a `finally` concept that you can stick this line